### PR TITLE
AWS Personalize: Revamp and add connection setting

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Revamped doc and added connection setting

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by utsabc — APPROVED** [review]

File: docs/destinations/analytics/aws-personalize.mdx [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+[Amazon Personalize](https://aws.amazon.com/personalize/), also known as **AWS Personalize**, is a machine learning service by Amazon. It enables you to create high-quality content recommendations, personalized product and marketing promotions, and much more.
 
-RudderStack lets you configure AWS Personalize as a destination to which you can seamlessly send your event data for personalized recommendation and effective product marketing.
+RudderStack supports AWS Personalize as a destination where you can send your event data seamlessly.
 
 <div class="successBlock">
 
-**Find the open-source transformer code for this destination in our** [**GitHub repo**](https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/personalize)**.**
+Find the open-source transformer code for this destination in our [GitHub repository](https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/personalize).
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/aws-personalize.mdx:14]:
```suggestion
Find the open source transformer code for this destination in the [GitHub repository](https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/personalize).
```

File: docs/destinations/analytics/aws-personalize.mdx [hunk 2]
```diff
@@ -32,31 +30,27 @@ Before configuring your source and destination on the RudderStack app, check whe
...
-Once you have confirmed that the source platform supports sending events to AWS Personalize, perform the steps below:
+Once you have confirmed that the source platform supports sending events to AWS Personalize, follow these steps:
 
-* Generate a Tracking ID by following the instructions [**here**](https://github.com/rudderlabs/rudder-transformer/tree/destination_personalize/v0/personalize/scripts).
+1. Generate a Tracking ID by following these [instructions](https://github.com/rudderlabs/rudder-transformer/tree/destination_personalize/v0/personalize/scripts).
 
 <div class="successBlock">
 
-**Keep this Tracking ID handy**. It is required to configure AWS Personalize as a destination.
+Keep this Tracking ID handy. It is required to configure AWS Personalize as a destination.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/aws-personalize.mdx:42]:
```suggestion
Keep the Tracking ID handy. It is required to configure AWS Personalize as a destination.
```

File: docs/destinations/analytics/aws-personalize.mdx [hunk 3]
```diff
@@ -32,31 +30,27 @@ Before configuring your source and destination on the RudderStack app, check whe
...
+To know more about the difference between cloud mode and device mode in RudderStack, refer to the <a href="https://rudderstack.com/docs/rudderstack-cloud/rudderstack-connection-modes/">RudderStack Connection Modes</a> guide.
 </div>
 
-Once you have confirmed that the source platform supports sending events to AWS Personalize, perform the steps below:
+Once you have confirmed that the source platform supports sending events to AWS Personalize, follow these steps:
 
-* Generate a Tracking ID by following the instructions [**here**](https://github.com/rudderlabs/rudder-transformer/tree/destination_personalize/v0/personalize/scripts).
+1. Generate a Tracking ID by following these [instructions](https://github.com/rudderlabs/rudder-transformer/tree/destination_personalize/v0/personalize/scripts).
 
 <div class="successBlock">
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/aws-personalize.mdx:40]:
```suggestion
<div class="infoBlock">
```

File: docs/destinations/analytics/aws-personalize.mdx [hunk 4]
```diff
@@ -32,31 +30,27 @@ Before configuring your source and destination on the RudderStack app, check whe
...
 
-* Next, go to the [**RudderStack dashboard**](https://app.rudderstack.com/), and configure the source.
-* From the list of destinations, select **AWS Personalize**. Assign a name to your destination and then click on **Next**. You should then see the following screen:
-
-<!--![Destination Settings for AWS Personalize](../../.gitbook/assets/personalize.png)-->
-
-<img src="../../assets/personalize.png" alt="Destination Settings for AWS Personalize" />
-<span class="imageTitle">Destination Settings for AWS Personalize</span>
+2. From your [RudderStack dashboard](https://app.rudderstack.com/), add the source. Then, from the list of destinations, select **AWS Personalize**.
+3. Assign a name to your destination and click on **Next**.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/aws-personalize.mdx:46]:
```suggestion
3. Assign a name to your destination and click on **Continue**.
```

File: docs/destinations/analytics/aws-personalize.mdx [hunk 5]
```diff
@@ -32,31 +30,27 @@ Before configuring your source and destination on the RudderStack app, check whe
...
 
-* Next, in the **Connection Settings** page, fill all the fields with the relevant information and click on **Next**.
+### Connection settings
 
-### Connection Settings
+To successfully configure AWS Personalize as a destination, configure the following settings:
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/aws-personalize.mdx:50]:
```suggestion
To successfully set up AWS Personalize as a destination, you need to configure the following settings:
```

File: docs/destinations/analytics/aws-personalize.mdx [hunk 6]
```diff
@@ -72,54 +66,48 @@ The following actions need to be attached to the Access Key while setting up the
...
 </div>
 
-* **Operational Choice**:
-  * **Personalize Events**: Choose the type of Personalize event you want to avail.
-  * **Map all the fields**: In this section, enter the **Schema Field** you have used to create the schema in AWS Personalize \(e.g. `USER_ID`, `TIMESTAMP`, `ITEM_ID`, etc.\). Also, enter the corresponding **Mapped Field** from which the value will be taken from your event payload.
-
 <div class="infoBlock">
 
-For more information on creating a schema in Personalize, refer to the <a href="https://docs.aws.amazon.com/personalize/latest/dg/data-prep-creating-datasets.html">Personalize documentation</a>.
+When using the <code class="inline-code">PutItems</code> operation, you need to provide the path to the corresponding <strong>Mapped Field</strong> corresponding to the <code class="inline-code">ITEM_ID</code> in your Personalize database schema.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/aws-personalize.mdx:86]:
```suggestion
When using the <code class="inline-code">PutItems</code> operation, you need to provide the path to the <strong>Mapped Field</strong> corresponding to the <code class="inline-code">ITEM_ID</code> present in your Personalize database schema.
```

File: docs/destinations/analytics/aws-personalize.mdx [hunk 7]
```diff
@@ -131,63 +119,52 @@ rudderanalytics.track("PRODUCT ADDED", {
...
-When using the `PutItems`, the **Schema Field** `ITEM_ID` must be mapped to a specific key inside the payload and you need to mention the path to the chosen key as the corresponding **Mapped Field**.
+When using the <code class="inline-code">PutItems</code>, the <strong>Schema Field</strong> <code class="inline-code">ITEM_ID</code> must be mapped to a specific key inside the payload and you need to mention the path to the chosen key as the corresponding <strong>Mapped Field</strong>.
 </div>
 
 In the above example, if you map `ITEM_ID` to the payload field `X`, the corresponding **Mapped Field** will be `properties.X`.
 
 <div class="infoBlock">
 
-For any other **Schema Field** in your `ITEMS` dataset, it is **not** advised to specify the path - only the field name will be sufficient.
+For any other <strong>Schema Field</strong> in your <code class="inline-code">ITEMS</code> dataset, it is <strong>NOT</strong> advised to specify the path - only the field name will be sufficient.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/aws-personalize.mdx:129]:
```suggestion
For any other <strong>Schema Field</strong> in your <code class="inline-code">ITEMS</code> dataset, it is <strong>not</strong> recommended to specify the path - only the field name is sufficient.
```

File: docs/destinations/analytics/aws-personalize.mdx [hunk 8]
```diff
@@ -131,63 +119,52 @@ rudderanalytics.track("PRODUCT ADDED", {
...
-<span class="imageTitle">Dashboard Configuration Based On The Above Example</span>
 
 The following image shows an example of the dashboard configuration for `PutEvents`:
 
-<!--![Dashboard Configuration Based On The Above Example](../../.gitbook/assets/putEvents.png)-->
-
 <img src="../../assets/putEvents.png" alt="Dashboard Configuration Based On The Above Example" />
-<span class="imageTitle">Dashboard Configuration Based On The Above Example</span>
 
 ## Identify
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/aws-personalize.mdx:145]: We should move the `identify` section before `track`.

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**